### PR TITLE
One to many

### PR DESCRIPTION
**Pull request by sandrus88**

### CODE REVIEW

File: src/main/java/org/sg/entities/ExamEntity.java [hunk 1]
```diff
@@ -15,25 +16,25 @@
 	
 	@Id
     @Column(name = "ID")
-	private Integer id;
+	private String id;
```
Comment (ermalaliraj) [inline src/main/java/org/sg/entities/ExamEntity.java]: Any specific reason to keep the id as string? We can keep the same approach of incremental int

File: src/main/java/org/sg/entities/ExamEntity.java [hunk 2]
```diff
@@ -82,11 +91,16 @@ public boolean equals(Object o) {
 		return true;
 	}
 	
+//	@Override
+//	public int hashCode() {
```
Comment (ermalaliraj) [inline src/main/java/org/sg/entities/ExamEntity.java]: If you have an equal() you need an hashCode() too.  For the same fields.
ps: equality of an exam is defined on: course, student, date, mark (in equals() I see only mark and date)

File: src/test/java/org/sg/test/ExamsTest.java [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+	CourseDao courseDao = new CourseDaoImpl();
+
+	@Test
+	public void test_get_exams_fromStudent() {
+		final Integer studentId = 3;
+		StudentEntity studentEntity = studentDao.get(studentId);
+		logger.info(studentEntity);
+		assertNotNull(studentEntity);
+
+		List<ExamEntity> exams = studentEntity.getExams();
```
Comment (ermalaliraj) [inline src/test/java/org/sg/test/ExamsTest.java]: you will have NPE if exam is null.

**Review by ermalaliraj — CHANGES_REQUESTED** [review]

File: src/main/java/org/sg/entities/ExamEntity.java (hunk 1 above)
Comment (sandrus88) [reply src/main/java/org/sg/entities/ExamEntity.java]: done

File: src/main/java/org/sg/entities/ExamEntity.java (hunk 2 above)
Comment (sandrus88) [reply src/main/java/org/sg/entities/ExamEntity.java]: done

File: src/test/java/org/sg/test/ExamsTest.java (hunk 3 above)
Comment (sandrus88) [reply src/test/java/org/sg/test/ExamsTest.java]: done

**Pull request merged**